lib/openapi3.js

```diff
@@ -44,18 +44,18 @@ function convertToToc(source,data) {
                     sMethodUniqueName = method.operation.summary;
                 }
                 method.slug = sMethodUniqueName.toLowerCase().split(' ').join('-'); // TODO {, } and : ?
-                var tagName = data.translations.defaultTag;
-                var tagDescription = '';
-                if (method.operation.tags && method.operation.tags.length > 0) {
-                    var tagData = getTagGroup(method.operation.tags[0], data.options.tagGroups);
-                    tagName = tagData.name;
-                    tagDescription = tagData.description;
+                if (typeof method.operation.tags === 'undefined' || !method.operation.tags.length) {
+                    method.operation.tags = [data.translations.defaultTag];
                 }
-                if (!resources[tagName]) {
-                    resources[tagName] = { count: 0, methods: {}, description: tagDescription};
+                for (let t in method.operation.tags) {
```

Comment:
If I read correctly, `method.operation.tags` is an array, so `method.operation.tags.forEach(...)` would make more sense.